app/controllers/application_controller.rb

```diff
@@ -1,3 +1,17 @@
+# frozen_string_literal: true
+
 class ApplicationController < ActionController::API
-        include DeviseTokenAuth::Concerns::SetUserByToken
+  include DeviseTokenAuth::Concerns::SetUserByToken
+  before_action :check_for_unpermitted_params
+  rescue_from ActionController::UnpermittedParameters, with: :unpermitted_parameters
+
+  private
+
+  def check_for_unpermitted_params
+    ActionController::Parameters.action_on_unpermitted_parameters = :raise
+  end
+
+  def unpermitted_parameters(e)
+    render json: { message: e.message }, status: 422
+  end
```

Comment:
I think I'm missing something. Is there a request spec for this functionality. What exactly are you achieving with this?

Reply:
no not needed, corpse ....